packages/plugins/response-cache-redis/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@envelop/response-cache-redis",
+  "version": "0.1.0",
```

Comment:
You can set this to `0.0.0` and create a changeset for the initial release. The changeset action will then take care of bumping the version.